app/src/main/res/values/strings.xml

```diff
@@ -34,5 +34,7 @@
     <string name="title_event_detail">Event Detail</string>
     <string name="subtitle_organizer">Organizer: %s</string>
     <string name="event_thumbnail">Event Thumbnail</string>
+    <string name="text_empty">Nothing to show here</string>
```

Comment:
But this is a generic layout, how do you plan to bind the message to event and detail layout. Setting text on load is a bad idea IMO. A generic text will be good as the layout can just be shown without caring to bind a specific message. "No items in this section" perhaps?